tools/gfx/renderer-shared.h

```diff
@@ -1372,10 +1374,7 @@ class RendererBase : public IDevice, public IShaderCacheStatistics, public Slang
     SlangContext slangContext;
     ShaderCache shaderCache;
 
-    // TODO: These should be removed when ShaderCacheIndex is ready to be integrated. ShaderCacheIndex
-    // will be responsible for keeping track of the underlying filesystem for the cache.
-    ISlangFileSystem* shaderCacheFileSystem = nullptr;
-    ComPtr<ISlangMutableFileSystem> mutableShaderCacheFileSystem = nullptr;
+    RefPtr<PersistentShaderCache> cacheIndex = nullptr;
```

Comment:
Just name it to `persistentShaderCache`.

Reply:
Changed locally, will be in next push